dev-tools/ch-dev/src/main.rs

```diff
@@ -43,10 +43,13 @@ enum ChDevCmd {
 #[derive(Clone, Debug, Args)]
 struct ChRunArgs {
     /// The HTTP port on which the server will listen
-    #[clap(short, long, default_value = "8123", action)]
-    port: u16,
+    #[clap(short = 'H', long, default_value_t = 8123, action)]
```

Comment:
Yeah, `clap` allows that just fine. I didn't do that because `ch-dev` doesn't currently depend on `omicron-common`, and I was hesitant to add such a large crate. I should have dug further though, since `omicron-common` _is_ in the dependency tree already, just not an immediate dep.

I'll change this and the other port too.

Reply:
Done in ffdcaa73a

Reply:
Nice, thanks!